.goreleaser_ui.yaml

```diff
@@ -11,6 +11,8 @@ builds:
       - amd64
     ldflags:
       - -s -w -X github.com/netbirdio/netbird/version.version={{.Version}} -X main.commit={{.Commit}} -X main.date={{.CommitDate}} -X main.builtBy=goreleaser
+    tags:
+      - legacy_appindicator
```

Comment:
Building the app with this tag will have any effect on the Debian packages?

Reply:
So far, there have been no observed effects. I did tests on 2 deb distros:
- Ubuntu:  works without installing any dependecies
- debian: works but I had to install [appindicator](https://extensions.gnome.org/extension/615/appindicator-support/) to work properly